src/System.Diagnostics.PerformanceCounter/pkg/System.Diagnostics.PerformanceCounters.pkgproj

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
You should rename this pkgproj to System.Diagnostics.PerformanceCounter without the "s". That will fix your cannot find description error.

Reply:
aha!